### PR TITLE
Mf dev 2

### PR DESCRIPTION
**Pull request by mfronheiser**
add database connections

### CODE REVIEW

File: server/Middleware/footprintsMiddleware.js [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+
+footprintsMiddleware.getCityId = async (req, res, next) => {
+  //   const { cityName } = req.body;
+
+  const answer = {
+    cityName: 'Philadelphia',
+    // brewName: '2nd Story Brewing Company'
+  }; //=>>>> CHANGE TO RES.BODY
+
+  const citytestName = 'Philadelphia';
```
Comment (Eliza612) [inline server/Middleware/footprintsMiddleware.js:13]: read from res.body?

File: server/models/footprintsModels.js [hunk 2]
```diff
@@ -1,16 +1,31 @@
...
+  // sets the name of the DB that our collections are part of
+  dbName: 'Brew',
+});
+
 const Schema = mongoose.Schema;
 
 const footprintsSchema = new Schema({
-  userId: { type: String, default: "user1" },
+  userId: { type: String, default: 'user1' },
   restaurant_name: { type: String, required: true },
```
Comment (Eliza612) [inline server/models/footprintsModels.js:19]: restaurant name may need to change to brewery name

**Review by Eliza612 — CHANGES_REQUESTED** [review]
Looks good! Just minor comments

**Comment by mfronheiser** [comment]
I believe I updated these changes here?

**Pull request closed without merging**